identity-iota-core/src/diff/mod.rs

```diff
@@ -0,0 +1,11 @@
+// Copyright 2020-2022 IOTA Stiftung
+// SPDX-License-Identifier: Apache-2.0
+
+pub use self::diff_iota_document::DiffIotaDocument;
+pub use self::diff_iota_document_metadata::DiffIotaDocumentMetadata;
+pub use self::diff_message::DiffMessage;
+
+mod diff_iota_did;
+mod diff_iota_document;
+mod diff_iota_document_metadata;
+mod diff_message;
```

Comment:
Aside: consider feature-gating all the diff structs and methods in `identity-iota-core`. Optionally in this PR or needs an issue.

Reply:
issue #710